services/account-without-juggler/index.ts

```diff
@@ -1,39 +1,13 @@
-import { Application, Server } from 'loopback-next/packages/core';
+import { Application } from '@loopback/core';
 import { AccountController } from './controllers/AccountController';
 import { AccountRepository } from './repositories/account';
 
-class AccountMicroservice extends Application {
-  private _startTime: Date;
+const app = new Application();
 
-  constructor() {
-    super();
-    const app = this;
-    app.controller(AccountController);
-    app.bind('repositories.account').toClass(AccountRepository);
-    app.bind('servers.http.enabled').to(true);
-    app.bind('servers.https.enabled').to(true);
-  }
+app.controller(AccountController);
+app.bind('http.port').to(3001);
+app.bind('repositories.account').toClass(AccountRepository);
 
-  async start() {
-    this._startTime = new Date();
-    const server = new Server(this, { port: 3001 });
-    server.bind('applications.accounts').to(this);
-    return server.start();
-  }
+app.start();
 
-  info() {
-    const uptime = Date.now() - this._startTime.getTime();
-    return { uptime: uptime };
-  }
-}
-
-async function main(): Promise<void> {
-  const app = new AccountMicroservice();
-  await app.start();
-  console.log('Application Info:', app.info());
-}
-
-main().catch(err => {
-  console.log('Cannot start the app.', err);
-  process.exit(1);
-});
+console.log('Application started on port:', app.getSync('http.port'));
```

Comment:
This message should wait until the promise returned by `app.start()` is resolved. When `http.port` is set to `0`, the value is updated to the real port when after the app starts listening. If your console log does not wait for `listen` to finish, then you may print incorrect port number.

Same comment applies to other app files changed by this pull request.